.github/scripts/generate-prdoc.py

```diff
@@ -110,27 +110,33 @@ def yaml_multiline_string_presenter(dumper, data):
 	yaml.add_representer(str, yaml_multiline_string_presenter)
 
 # parse_args is also used by cmd/cmd.py
-def setup_parser(parser=None):
+# if pr_required is False, then --pr is optional, as it can be derived from the PR comment body
+def setup_parser(parser=None, pr_required=True):
+	allowed_audiences = ["runtime_dev", "runtime_user", "node_dev", "node_operator"]
 	if parser is None:
 		parser = argparse.ArgumentParser()
-	parser.add_argument("--pr", type=int, required=True, help="The PR number to generate the PrDoc for."	)
-	parser.add_argument("--audience", type=str, default="TODO", help="The audience of whom the changes may concern.")
-	parser.add_argument("--bump", type=str, default="TODO", help="A default bump level for all crates.")
-	parser.add_argument("--force", type=str, help="Whether to overwrite any existing PrDoc.")
-	
+	parser.add_argument("--pr", type=int, required=pr_required, help="The PR number to generate the PrDoc for.")
+	parser.add_argument("--audience", type=str, nargs='*', choices=allowed_audiences, default=["todo"], help="The audience of whom the changes may concern. Example: --audience runtime_dev node_dev")
+	parser.add_argument("--bump", type=str, default="major", choices=["patch", "minor", "major", "silent", "ignore", "no_change"], help="A default bump level for all crates. Example: --bump patch")
```

Comment:
Would be nice if this could be taken from the CI check. Aka to take the `bump` as determined by the tool.

Reply:
yep, i also thought about it

Reply:
I think i'll try to do this within https://github.com/paritytech/prdoc/issues/46